user/telemetry.md

```diff
@@ -0,0 +1,78 @@
+# User telemetry
+
+_Introduced in v3.4.0_
```

Comment:
Might be worth mentioning that while it was introduced in 3.4 various things are being added all the time, and this is the latest list of stats.